lib/pluginmanager/update.rb

```diff
@@ -8,25 +8,28 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   def execute
     local_gems = gemfile.locally_installed_gems
 
-    if update_all? && !local_gems.empty?
-      error_plugin_that_use_path!(local_gems)
-    else
-      plugins_with_path = plugins_arg & local_gems
-      error_plugin_that_use_path!(plugins_with_path) if plugins_with_path.size > 0
+    if local_gems.size > 0
```

Comment:
Why not checking if there are local_gems with ``` if !local_gems.empty? ``` might be more intutive than checking for size > 0. Is there any reason for this change that I might be missing ?